packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -40,12 +37,7 @@ function LoginForm({
   );
   const { mutateAsync: onPasswordLogin } = useMutation(
     (params: ValidateLoginParams) => submitKratosFlow(params),
-    {
-      onSuccess: ({ data }) => {
-        const session = data as AuthSession;
-        onUpdateSession(session);
-      },
-    },
+    // { onSuccess: ({ data }) => {} }, // refetch boot after login
```

Comment:
Same here, are you tackling this seperate?

Reply:
Yes, it will be part of the held off PR for login and registration.